src/main/java/calculator/model/Delimiters.java

```diff
@@ -0,0 +1,25 @@
+package calculator.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Delimiters {
+
+    private List<String> delimiterList;
+
+    public Delimiters() {
+        this.delimiterList = new ArrayList<>();
+    }
+
+    public List<String> getDelimiterList() {
+        return delimiterList;
+    }
+
+    public void addDelimiter(String delimiter) {
+        this.delimiterList.add(delimiter);
+    }
+
+    public void addAllDelimiter(List<String> delimiters) {
+        this.delimiterList.addAll(delimiters);
+    }
```

Comment:
```suggestion
    public void addAllDelimiters(List<String> delimiters) {
        this.delimiterList.addAll(delimiters);
    }
```

여러 개 추가하는만큼 복수형은 어떨까요?